### PR TITLE
Improved the icons of community menu with officials icon

### LINKED ISSUE #8517: Improve the icons of community menu with official icons
The icons in the community menu are not looking good replacing them with official logos will make it look good.

Before:- 
before
After:-
after

I have already done this so if it looks like it will be a good pull request then I will do the pull request.

### PR DESCRIPTION
**Pull request by tanmayChakrawarty**
## Purpose / Description
The icons in the community menu are not looking good replacing them with official logos will make the UI more enhancing.

## Fixes
Fixes  #8517 

## Approach
Before:-
before

After:-
after

## How Has This Been Tested?
Tested on a physical device (Android 5.0).

## LICENCE
LICENSE:- https://github.com/Templarian/MaterialDesign/blob/master/LICENSE

## Other links related to icons
Github Repo:- https://github.com/Templarian/MaterialDesign
Reddit icon:- https://dev.materialdesignicons.com/icon/reddit
Discord icon:- https://dev.materialdesignicons.com/icon/discord
Facebook icon:- https://dev.materialdesignicons.com/icon/facebook
Twitter icon:- https://dev.materialdesignicons.com/icon/twitter

## Checklist
- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented on your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)

### CODE REVIEW

**Comment by mrudultora** [comment]
```
  /home/runner/work/Anki-Android/Anki-Android/AnkiDroid/src/main/res/drawable/ic_link_black_24dp.xml:1: Error: The resource R.drawable.ic_link_black_24dp appears to be unused [UnusedResources]
  <vector xmlns:android="http://schemas.android.com/apk/res/android"
  ^
  /home/runner/work/Anki-Android/Anki-Android/AnkiDroid/src/main/res/drawable/ic_mail_outline_black_24dp.xml:1: Error: The resource R.drawable.ic_mail_outline_black_24dp appears to be unused [UnusedResources]
  <vector xmlns:android="http://schemas.android.com/apk/res/android"
  ^
  /home/runner/work/Anki-Android/Anki-Android/AnkiDroid/src/main/res/drawable/ic_message_black_24dp.xml:1: Error: The resource R.drawable.ic_message_black_24dp appears to be unused [UnusedResources]
  <vector xmlns:android="http://schemas.android.com/apk/res/android"
  ^
```
As you have removed the use of previous icons, please delete them from the project too. The drawables to be removed can be found in above message.

**Review by mrudultora — COMMENTED** [review]
Is there any reason you put those svg's in `drawable-xxhdpi` ?

**Comment by tanmayChakrawarty** [comment]
> Is there any reason you put those svg's in `drawable-xxhdpi` ?

As I get these vector assets from the given links when I include them,they are by default showing `drawable-xxhdpi`

**Comment by infinyte7** [comment]
Is it good to have color for icons with their original icons?

**Review by mikehardy — CHANGES_REQUESTED** [review]
I like that we have consistent colors, I think it is pleasing, I think it will look jarring if we use the official brand colors

I would not just put them in xxhdpi, I would put them right next to where all the other SVGs are, or where the ones being replaced were

The original icons are now unused and should be deleted, until then, the quality checks will fail: 

https://github.com/ankidroid/Anki-Android/pull/8519/checks?check_run_id=2297601290#step:8:70

I like this in general though, it looks really nice

File: AnkiDroid/src/main/res/drawable-xxhdpi/discord.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!-- drawable/discord.xml -->
```
Comment (david-allison-1) [inline AnkiDroid/src/main/res/drawable-xxhdpi/discord.xml:1]: @mikehardy Do we want the license in each file for these?

Comment (tanmayChakrawarty) [reply AnkiDroid/src/main/res/drawable-xxhdpi/discord.xml:1]: I have added the license in each XML file.

**Comment by mrudultora** [comment]
@tanmayChakrawarty It would be good if you put those drawables in `drawable` folder, rather than `drawable-xxhdpi` ;-)

**Comment by tanmayChakrawarty** [comment]
> @tanmayChakrawarty It would be good if you put those drawables in `drawable` folder, rather than `drawable-xxhdpi` ;-)

OK, I will do that. :-)

**Review by mikehardy — APPROVED** [review]
This looks good to me thanks @tanmayChakrawarty !
I restarted the emulator tests, they failed for an unrelated reason

**Review by david-allison-1 — APPROVED** [review]
Looks good to me!

**Pull request merged**